package.json

```diff
@@ -68,10 +68,10 @@
     "@uniswap/permit2-sdk": "^1.2.0",
     "@uniswap/redux-multicall": "^1.1.8",
     "@uniswap/router-sdk": "^1.3.0",
-    "@uniswap/sdk-core": "^3.0.1",
-    "@uniswap/smart-order-router": "^2.10.0",
+    "@uniswap/sdk-core": "^4.0.0",
+    "@uniswap/smart-order-router": "^3.6.0",
```

Comment:
This requires ethers@^5.7.2, so ethers should be updated as well.
This will likely require some of our users to upgrade their ethers.